lxd/daemon.go

```diff
@@ -318,13 +318,13 @@ func StartDaemon(listenAddr string) (*Daemon, error) {
 
 	/* Create required paths */
 	d.lxcpath = shared.VarPath("lxc")
-	err = os.MkdirAll(shared.VarPath("/"), 0755)
+	err = os.MkdirAll(shared.VarPath("lxc"), 0755)
```

Comment:
How about just using d.lxcpath here since we've done the VarPath dance already a line earlier?